src/main/webapp/app/overview/course-overview.component.html

```diff
@@ -12,19 +12,44 @@
                 <div class="sidebar-container d-flex h-100 justify-content-between flex-column" [ngClass]="{ collapsed: isNavbarCollapsed }">
                     <!-- NavItems -->
                     <div>
-                        <div class="d-flex p-3 align-items-center text-decoration-none" [title]="course?.title">
+                        <div #container id="container" class="d-flex p-3 align-items-center text-decoration-none" [title]="course?.title" (click)="onContainerClick()">
                             @if (course && course.courseIcon) {
                                 <div>
-                                    <jhi-secured-image [src]="course.courseIcon" />
+                                    <jhi-secured-image id="courseIcon" [src]="course.courseIcon" />
                                 </div>
                             } @else {
-                                <div class="course-circle d-flex align-items-center justify-content-center">
+                                <div id="courseCircle" class="course-circle d-flex align-items-center justify-content-center">
                                     <span class="fs-4">{{ course?.title | slice: 0 : 1 }}</span>
                                 </div>
                             }
                             @if (!isNavbarCollapsed) {
                                 <div id="test-course-title" class="h6 mb-0 fw-bold text-body auto-collapse">{{ course?.title }}</div>
                             }
+                            <div id="courseDropdownButton" class="d-inline-flex position-relative m-2">
+                                <fa-icon [icon]="faChevronDown" />
+                            </div>
```

Comment:
The chevron should indicate whether the list is collapsed or uncollapsed, this can be achieved by CSS classes - we do already have some shared classes for that.

Reply:
adjusted it